packages/web3-shared/base/src/specs/index.ts

```diff
@@ -394,6 +394,18 @@ export interface NonFungibleCollection<ChainId, SchemaType> {
     source?: SourceType
     assets?: Array<NonFungibleAsset<ChainId, SchemaType>>
     socialLinks?: SocialLinks
+    floorPrices?: Array<{
+        marketplace_id: LiteralUnion<'blur' | 'looksrare' | 'opensea' | 'x2y2'>
```

Comment:
SourceType

Reply:
It's marketplace, a little different from SourceType